README.md

```diff
@@ -86,20 +84,126 @@ code because of all the basic column-renamings and the full listing of health co
+
+#### CSLB Schema Design
+* A single `cslb` table which contains a single CSLB record per dog per year. 
+* The table includes the 16 CSLB question responses and a foreign key to the dog table. 
+
+
+## Environmental Data
+Environmental Data in the Dog Aging Project is published monthly to RedCap for each dog using the addresses
+collected in the HLES survey. It seeks to capture a snapshot of key environmental factors that may affect a 
+dog's health over time so it can be used in conjunction with the other datasets. All fields from environmental 
+data are calculated using the HLES primary and secondary addresses.
+
+#### Environmental Extraction Criteria
+* Pull all records from each monthly arm for each address: *"annual_{MMMyyyy}_arm_1"*, *"annual_{MMMyyyy}_secondary_arm_1* 
+* where _baseline_complete_ is marked as complete.
+
+#### Environment Schema Design
+Environmental data is modeled closely on what the DAP provided us, with a proto-schema which we
+converted to a repo schema. We decided to break the larger table up into smaller fragments: 
+_geocoding, census, pollutants, temperature_precipitation, and walkability_ variables.
+
+* A single `environment` table which contains a single Environment record per dog per month per address. 
+* The table includes all environmental variables and a foreign key to the dog table. 
+* Rows in the final Environment table will be unique on _dog_id_ and _address_month_year_.
+
+
+## Sample Data
+Sample Data in the Dog Aging Project is populated as different cohorts of participants are sent different 
+sample kits. Our extraction of this data seeks to capture the linkage between study_id (_dog_id_) and 
+sample_id as well as some other metadata about the sample.
+
+#### Sample Extraction Criteria
+* Pull all records from the baseline arm (*baseline_arm_1*) 
+* where _k1_tube_serial_ and _k1_rtn_tracking_date_ are populated.
+
+#### Sample Schema Design
+* A single `sample` table which contains multiple samples per dog.
+* Simple lookup table with 6 fields and a foreign key to the dog table.
+* Rows in the final Environment table will be unique on _dog_id_ and _sample_id_.
+
+
+## Label Mapping Table
+We have a manually maintained lookup table that we are constructing using the Terra JSON schema for the DAP tables and 
+the DAP data dictionaries provided from RedCap. The table contains a row for each field in the the final dog aging tables.
+The mapping table includes fields to capture information about the upstream source of data, datatypes, and most importantly,
+the mapping of raw values ot value labels for RedCap survey questions with *radio* responses.
+
+We are looking forward to building some tooling that will be able to
+parse a proto-schema provided by DAP into usable code for ingest in the form of a Terra JSON Schema output.
+
+
+#### Creating Outfiles 
+DAP's long term future would include an official TDR<->Workspace integration but we are currently still building that out.
```

Comment:
One more nit: we are not in the process of building that out (sorry if I led you astray with a previous comment). To avoid any potential confusion let's say:
```suggestion
The DAP ingest's long term roadmap includes a TDR<->Workspace integration. Until that is integration is live...
```